SpywareFilter/sections/tracking_servers.txt

```diff
@@ -43,6 +43,8 @@
 ||visilabs.net^$third-party
 !
 ||heymatic.com^$third-party
+||pix.impdesk.com^$third-party
```

Comment:
I don't see these requests on this site, could you make a screenshot where these requests are visible?

Reply:
If I am not wrong, it's a preconnect element. I have also found this in Easyprivacy after enabling it.

![image](https://user-images.githubusercontent.com/76908410/113140887-971ef600-925b-11eb-80c9-a717f68683e3.png)

Reply:
Ok, I see EasyPrivacy contains both domains